docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -33,7 +33,7 @@ We conduct testing and ensure compatibility against the following OpenShift vers
 | 4.13.x            | November 17, 2024                                                                 |
 
 :::caution
-Compatibility is not guaranteed for OpenShift versions no longer supported by Red Hat, as per the End of Support Date. For more information, refer to the [Red Hat OpenShift Container Platform Life Cycle Policy](https://access.redhat.com/support/policy/updates/openshift).
+Camunda supports OpenShift versions that fall within the Red Hat General Availability, Full Support, and Maintenance Support life cycle phases. For more information, refer to the [Red Hat OpenShift Container Platform Life Cycle Policy](https://access.redhat.com/support/policy/updates/openshift).
```

Comment:
```suggestion
Camunda 8 only supports OpenShift versions in Red Hat General Availability, Full Support, and Maintenance Support life cycle phases. For more information, refer to the [Red Hat OpenShift Container Platform Life Cycle Policy](https://access.redhat.com/support/policy/updates/openshift).
```
Wording lgtm, just an idea for rewording here but feel free to ignore as I'm not sure it's any better - I did add Camunda 8 in however as I wasn't sure if we should clarify whether we mean Camunda (the company) supports..., or Camunda 8 (the software) supports...

Reply:
I like these suggestions - I've gone ahead and added them in, thank you!